### DISCUSSION TITLE
Know this command? (`docker network update`)

### DISCUSSION THREAD

**Discussion by DaniHRE** [moby/moby#45138] [Q&A] [answered]
Hello Guys.
Is `docker network update` a real docker command?
If it's true. how can i use this. cause my docker don't have this command available

**Comment by cpuguy83** [comment, marked as answer]
This is not a real command.
Networks are considered immutable except that you can attach/detach containers from them.

**Reply by DaniHRE** [reply]
ChatGPT cheating me everyday... lol

**Reply by thaJeztah** [reply]
Ha! Yes, it definitely does some wishful thinking at times. FWIW, we did discuss it at some point, e.g. for situations where you'd want to change the ip-range for an existing network (this was a discussion within scope of a "swarm"), but it'd be very complicated to do so if the network is in use. So, "maybe ... one day?"
